__tests__/app.test.js

```diff
@@ -79,14 +79,70 @@ describe("NC-News", () => {
         });
     });
   });
+  describe("GET /api/articles/:article_id", () => {
+    test("It should respond with an article object by article_id", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article).toBeInstanceOf(Object);
+          expect(body.article.article_id).toBe(1);
+        });
+    });
+    test("It should have the correct keys", () => {
+      return request(app)
+        .get("/api/articles/2")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article).toHaveProperty("author", expect.any(String));
+          expect(body.article).toHaveProperty("title", expect.any(String));
+          expect(body.article).toHaveProperty("article_id", expect.any(Number));
+          expect(body.article).toHaveProperty("body", expect.any(String));
+          expect(body.article).toHaveProperty("topic", expect.any(String));
+          expect(body.article).toHaveProperty("created_at", expect.any(String));
+          expect(body.article).toHaveProperty("votes", expect.any(Number));
+          expect(body.article).toHaveProperty(
+            "article_img_url",
+            expect.any(String)
+          );
+        });
+    });
+  });
+  describe("GET /api/articles/:article_id/comments", () => {
+    test("It should respond with an array of comments", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.comments).toBeInstanceOf(Array);
+          expect(body.comments).toHaveLength(11);
+        });
+    });
```

Comment:
This is a good test for this ticket, though we'll need a few more for the endpoint /api/articles/:article_id/comments
400 - invalid article ID
404 - article not found
We'll also need to test that the comments are in date order (using jest-sorted)
We'll also need to test that when we look for the comments where the article exists but there are no comments, we get an empty array rather than a 404 error 🔍